.github/CODEOWNERS

```diff
@@ -1,4 +1,6 @@
 * @youtube/cobalt-3p-repository-owners
 /.github/  # no owners
 /cobalt/  # no owners
+/chrobalt/  # no owners
```

Comment:
This one should be removed now ! Brian renamed chrobalt->cobalt

Reply:
Done